pkgs/development/ocaml-modules/gstreamer/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, buildDunePackage, fetchFromGitHub, dune-configurator, pkg-config, glib, gst_all_1 }:
+{ lib, stdenv, buildDunePackage, fetchFromGitHub, dune-configurator, AppKit, Foundation, pkg-config, glib, gst_all_1 }:
```

Comment:
Does I profit of this PR to split deps on each line and sort them ?

Reply:
I would prefer it to be in a separate PR whose name starts with treewide.